client/my-sites/stats/info-panel/index.jsx

```diff
@@ -112,8 +112,8 @@ class StatsInfoPanel extends React.PureComponent {
 									href="http://wordpress.com/support/stats/#search-engine-terms"
 								>
 									<Gridicon icon="info-outline" />
-									{ this.props.translate( 'About Search Terms', {
-										context: 'Stats: search terms info module documentation link',
+									{ this.props.translate( 'About search terms', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.